pom.xml

```diff
@@ -41,6 +41,21 @@
                  <goals>deploy</goals>
              </configuration>
           </plugin>
+          <plugin>
+            <groupId>org.codehaus.gmaven</groupId>
+            <artifactId>gmaven-plugin</artifactId>
+                <version>1.5-jenkins-3</version>
+            <configuration>
+                <providerSelection>1.7</providerSelection>
+            </configuration>
+            <dependencies>
+                <dependency>
+                    <groupId>org.codehaus.gmaven.runtime</groupId>
+                    <artifactId>gmaven-runtime-1.7</artifactId>
+                    <version>1.3</version>
+                </dependency>
+            </dependencies>
+          </plugin>
```

Comment:
Ah, I did not know there was a `HtmlPublisherTest.groovy`. Worth adding a `TODO` comment to this effect, since newer versions of the parent POM do use `1.5-jenkins-3`.